apps/modernization-api/src/test/java/gov/cdc/nbs/patient/PatientMother.java

```diff
@@ -113,27 +111,30 @@ public void deleted(final PatientIdentifier identifier) {
         new PatientCommand.Delete(
             identifier.id(),
             this.settings.createdBy(),
-            this.settings.createdOn()
-        ),
-        id -> 0
-    );
+            this.settings.createdOn()),
+        id -> 0);
+  }
+
+  public void superseded(final PatientIdentifier identifier) {
+    Person patient = managed(identifier);
+
+    patient.setRecordStatusCd(RecordStatus.SUPERCEDED);
+    this.entityManager.persist(patient);
```

Comment:
nit:  The class is annotated with `@Transactional` so any changes made to the `Person` entity will be persisted automatically making the call to `persist` redundant.